### PR TITLE
[IMP] purchase_stock, stock: Remove Security Lead Time from PO's picking deadline

### PR DESCRIPTION
**Pull request by clesgow**
Remove the Purchase Security Lead Time from the computing of the purchase order's picking deadline to better reflect that the shipment is actually meant to arrive earlier than date + security lead time.

Meaning if we have a Purchase Security Lead Time of 30 days, when a reordering rule is triggered, the picking for the created purchase order won't be meant to come in 30+ days, but as soon as possible.

Task-2656397
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/stock/models/stock_orderpoint.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def _domain_product_id(self):
     product_uom_name = fields.Char(string='Product unit of measure label', related='product_uom.display_name', readonly=True)
     product_min_qty = fields.Float(
         'Min Quantity', digits='Product Unit of Measure', required=True, default=0.0,
-        help="When the virtual stock equals to or goes below the Min Quantity specified for this field, Odoo generates "
+        help="When the virtual stock goes below the Min Quantity specified for this field, Odoo generates "
```
Comment (amoyaux) [inline addons/stock/models/stock_orderpoint.py:75]: Don't forget to adapt the .pot

**Comment by amoyaux** [comment]
Ok for me, just update the .pot for the settings also

File: addons/stock/models/stock_orderpoint.py [hunk 2]
```diff
@@ -551,3 +551,6 @@ def _procure_orderpoint_confirm(self, use_new_cursor=False, company_id=None, rai
 
     def _post_process_scheduler(self):
         return True
+
+    def _get_orderpoint_procurement_date(self):
```
Comment (amoyaux) [inline addons/stock/models/stock_orderpoint.py:555]: I'm not sure about that it seems a bit risky IMO. I would rather add a context key to bypass company po_lead in the _compute_lead_days. What do you think of it?

Comment (amoyaux) [reply addons/stock/models/stock_orderpoint.py:555]: No bad idea, keep it like currently

**Comment by clesgow** [comment]
Updated the PR as requested @amoyaux

**Comment by amoyaux** [comment]
robodoo r+ rebase-ff

**Pull request closed without merging**